packages/a2aprotocol/pyproject.toml

```diff
@@ -8,9 +8,10 @@ requires-python = ">=3.12"
 repository = "https://github.com/microsoft/teams.py"
 keywords = ["microsoft", "teams", "ai", "bot", "agents"]
 license = "MIT"
-classifiers = ["Private :: Do Not Upload"]
 dependencies = [
     "a2a-sdk[all]>=0.3.7",
```

Comment:
btw what does [all] constitute?

Reply:
it downloads the optional dependencies as well, added this cause we have a couple dependencies in the "all extra" bundle in addition to the core package 
https://github.com/a2aproject/a2a-python?tab=readme-ov-file#-installation

Reply:
Yeah those don't seem helpful for us. Let's be intentional about what we pull